microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-105.md

```diff
@@ -0,0 +1,155 @@
+
+<!-- Title: Focus Mode: Improved location controls for DevTools, Activity Bar, and Quick View -->
+<!-- Subtitle: Focus Mode: Improved location controls for DevTools, Activity Bar, and Quick View. -->
+
+In Microsoft Edge 105, several improvements to location controls have been made in [Focus Mode](/devtools-guide-chromium/experimental-features/focus-mode), including a new option for changing the orientation of **Quick View**.
+
+The **Customize and control DevTools** (**...**) menu now directly provides options for changing the docking location of DevTools.  This menu also contains settings for where the **Activity Bar** is displayed.  The previous submenus have been replaced with toggle buttons.  Additionally, improved contrast means the option you have selected is more easily visible at a glance.
+
+Selecting the docking location in Focus Mode, in previous versions of Microsoft Edge:
+
+![Focus Mode Before](./devtools-105-images/before-docking-menu.png)
+
+Selecting the docking location in Focus Mode, in Microsoft Edge 105:
+
+![Focus Mode After](./devtools-105-images/after-docking-menu.png)
+
+You can now change the orientation of the **Quick View** panel as well.  To display **Quick View** vertically instead of horizontally, click the **Dock Quick View to the right** toggle button.  Clicking this toggle again returns **Quick View** to the horizontal orientation.
+
+![Docking Quick View to the right](./devtools-105-images/quickview-console.png)
+
+Returning **Quick View** to the horizontal orientation:
+
+![Docking Quick View to the bottom](./devtools-105-images/dock-quick-view-bottom.png)
+
+To minimize **Quick View** in either orientation, click the **Collapse Quick View** toggle button, or press `Esc`:
+
+![Minimizing Quick View](./devtools-105-images/focus-mode-improved-location-controls.png)
+
+See also:
+* [Simplify DevTools using Focus Mode](../../../experimental-features/focus-mode.md)
+
+
+<!-- ====================================================================== -->
+## Fix: Search in the Sources and Network tools now works as expected in Focus Mode
+
+<!-- Title: Fix: Search in the Sources and Network tools now works as expected in Focus Mode  -->
+<!-- Subtitle: Try Focus Mode, a new, more simplified, and streamlined UI for DevTools! -->
+
+In previous versions of Microsoft Edge, the Focus Mode experiment didn't properly showcase the **Search** feature in the **Network** tool or the **Search in all files** feature in the **Sources** tool.  In Microsoft Edge 105, these issues have been fixed.
+
+You can now search in the **Network** tool via the **Search** icon:
+
+![Networking Search](./devtools-105-images/networking-search.png)
+
+To search in all files in the **Sources** tool, right-click a window icon node or a cloud icon node in the **Page** section, and then select **Search in all files**.  Or right-click a folder icon node and then select **Search in folder**.  This displays the **Search** tool in **Quick View**:
```

Comment:
We can cut the **Search in folder** feature since this fix was specifically about **Search in all files**.

```suggestion
To search in all files in the **Sources** tool, right-click the `top` window or any domain in the **Page** section, and then select **Search in all files**. 
```